bio_market_store/store/views.py

```diff
@@ -208,29 +211,22 @@ def user_profile_personal_info(request):
 
 @login_required
 def user_profile_address(request):
-    user_profile = request.user
-    address, created = Address.objects.get_or_create(user=user_profile)
     if request.method == "POST":
-        address_form = AddressForm(request.POST, instance=address)

```

Comment:
Why did you replace AddressForm? 
You could add missing properties to the AddressForm.